### ISSUE TITLE
MemoryStore warning in production environment

### ISSUE THREAD

**Issue by VarunBatraIT** [iloire/watchmen#48]
```
run-web-server-23 (err): Warning: connect.session() MemoryStore is not
run-web-server-23 (err): designed for a production environment, as it will leak
run-web-server-23 (err): memory, and will not scale past a single process.
```

I am getting this warning when I use production environment. I hope things are ok?

**Comment by iloire** [comment]
Using ``MemoryStore`` is not recommended for production environments because you can potentially ended up with memory leaks if sessions are not properly cleaned up from memory.

Adding a redis based session store would be trivial but it needs to be done preferably with breaking the storage abstraction.

**Comment by iloire** [comment]
Fixed with 516e940cea96dbe706fdeb36737b262cf105c491
